lib/compat/wordpress-6.1/blocks.php

```diff
@@ -205,3 +205,44 @@ function gutenberg_block_type_metadata_multiple_view_scripts( $metadata ) {
 	return $metadata;
 }
 add_filter( 'block_type_metadata', 'gutenberg_block_type_metadata_multiple_view_scripts' );
+
+/**
+ * Register render template for core blocks if handling is missing in WordPress core.
+ *
+ * @since 6.1.0
+ *
+ * @param array $settings Array of determined settings for registering a block type.
+ * @param array $metadata Metadata provided for registering a block type.
+ *
+ * @return array Array of settings for registering a block type.
+ */
+function gutenberg_block_type_metadata_render_template( $settings, $metadata ) {
+	if ( empty( $metadata['render'] ) || isset( $settings['render_callback'] ) ) {
+		return $settings;
+	}
+
+	$template_path = wp_normalize_path(
+		realpath(
+			dirname( $metadata['file'] ) . '/' .
+			remove_block_asset_path_prefix( $metadata['render'] )
+		)
+	);
+
```

Comment:
We should probably add a check here to avoid fatalities in case the file is missing for some reason

```suggestion

	// Bail if the file does not exist.
	if ( ! file_exists( $template_path ) ) {
		return $settings;
	}
```

Reply:
Good catch, included.